Expensify/App
# [CP Staging] Fix greyed out workspace icon for workspace distance requests

### Details
Use `||` instead of `??` since ```allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${report?.policyID}`]?.avatar``` can return `''`


### Fixed Issues
$ https://github.com/Expensify/App/issues/32984
PROPOSAL: N/A 


### Tests

1. Create a NewDot workspace if you don't already have one.
2. Open the workspace chat for your workspace.
3. Press the +, Request Money, then Distance and complete a distance request.
4. Confirm the workspace avatar is visible in the conversation report preview


### Offline tests
N/A 

### QA Steps
Same as test steps 


### PR Author Checklist


### Screenshots/Videos
<details>
<summary>Android: Native</summary>



</details>

<details>
<summary>Android: mWeb Chrome</summary>



</details>

<details>
<summary>iOS: Native</summary>



</details>

<details>
<summary>iOS: mWeb Safari</summary>



</details>

<details open>
<summary>MacOS: Chrome / Safari</summary>
(reported on Web only) 

https://github.com/Expensify/App/assets/16747822/39ba6212-b7b3-4cff-b2f9-01137ed9feed

</details>

<details>
<summary>MacOS: Desktop</summary>



</details>
